Modules/Loadable/InteractiveTubesToTree/Logic/vtkSlicerInteractiveTubesToTreeLogic.cxx

```diff
@@ -487,54 +510,74 @@ ::deleteTubeFromSpatialObject(vtkMRMLSpatialObjectsNode* spatialNode, std::set<i
     it = tubeIDs.find(currTubeId);
     if(it !=  tubeIDs.end())
```

Comment:
VesselView is a mix of Qt, ITK, and VTK classes - and each has its own
style.  None of our commonly used automated style checkers can deal with
that mix...yet...

On Thu, Dec 31, 2015 at 11:33 AM, sumedhasingla <notifications@github.com>
wrote:

> In
> Modules/Loadable/InteractiveTubesToTree/Logic/vtkSlicerInteractiveTubesToTreeLogic.cxx
> <https://github.com/KitwareMedical/VesselView/pull/148#discussion_r48662232>
> :
>
> > @@ -487,54 +510,74 @@ ::deleteTubeFromSpatialObject(vtkMRMLSpatialObjectsNode* spatialNode, std::set<i
> >      it = tubeIDs.find(currTubeId);
> >      if(it !=  tubeIDs.end())
>
> As of now VesselView code doesn't check the formatting issues on Git
> commit. Apparently there are many space issues in the code. I will fix them
> in a different commit.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/KitwareMedical/VesselView/pull/148/files#r48662232>.
>



-- 
====================================================
Stephen R. Aylward, Ph.D.
Senior Director of Operations, North Carolina, Kitware, Inc.
http://www.kitware.com and http://www.aylward.org
(919) 969-6990 x300
====================================================